src/components/Create/components/pages/ReviewDeploy/components/RulesReview/RulesReview.tsx

```diff
@@ -21,7 +22,7 @@ export const RulesReview = () => {
       {isMobile ? (
         <MobileRulesReview />
       ) : (
-        <Row gutter={20}>
+        <Row gutter={20} className="gap-y-5">
```

Comment:
I think its more correct to use two separate rows here rather than let the row wrap around with span pushing

Reply:
In the spirit of anti-antd, i removed `row` and went with a wrapping flex \o/